### ISSUE TITLE
[v1.2] equation label generates (non-lethal) parse error

### ISSUE THREAD

**Issue by pkra** [zorkow/speech-rule-engine#122]
Reposting here.

Steps to reproduce.

a. `npm install pkra/mathjax-node-sre`
b. run

```javascript
const mjsre = require('../lib/main.js').typeset;  
const input = '\\begin{equation} \\text{Hello} \\tag{tag} \\end{equation}'
mjsre({math: input, format: "TeX", svgNode: true, speakText:true},function(result){
      console.log(result.svg);
});
```
c. console output:
  * `Parse Error: this.getFactory is not a function`
  * `<svg .... <desc ... >matrix ( Tag ) Hello</desc> ...</svg>`

**Comment by zorkow** [comment]
This works now in ```speech-rule-engine@2.0.0-beta```
However, I don't get any output for ```result.svg```. But for 
```mjsre({math: input, format: "TeX", svgNode: true, speakText:true},function(result){ console.log(result.mml); });```
I get
```<math xmlns="http://www.w3.org/1998/Math/MathML" display="block" alttext="StartLayout 1st Row  with Label left-parenthesis tag right-parenthesis EndLabel Hello EndLayout">
  <mtable displaystyle="true">
    <mlabeledtr>
      <mtd id="mjx-eqn-tag">
        <mtext>(tag)</mtext>
      </mtd>
      <mtd>
        <mtext>Hello</mtext>
      </mtd>
    </mlabeledtr>
  </mtable>
</math>
```

**Comment by zorkow** [comment]
Fixed in v2.0.0-beta.1.
